app/views/management/users/_erase_user_account.html.erb

```diff
@@ -1,9 +1,11 @@
-<%= link_to t("management.users.erase_account_link"), "#", class: "button hollow alert js-toggle-link", data: { "toggle-selector" => "#erase-account-form" } %>
+<%= link_to t("management.users.erase_account_link"), "#", class: "button hollow alert js-toggle-link edit-account-form", data: { "toggle-selector" => "#erase-account-form" },
```

Comment:
I would indent this element more like [this one here](https://github.com/AyuntamientoMadrid/consul/blob/master/app/views/budgets/index.html.erb#L41) to make it more readable.